### DISCUSSION TITLE
How can I `git clean -df`?

### DISCUSSION THREAD

**Discussion by zocker-160** [jesseduffield/lazygit#3756] [Q&A] [answered]
The discard changes option seems to only discard a single file / folder, is there a way to reset everything like `git clean -df` does?

**Comment by stefanhaller** [comment, marked as answer]
Hit `D` to bring up the `Reset...` menu; the "Discard untracked files" option does exactly that.

**Reply by zocker-160** [reply]
ahh it is called reset, thank you
